contents/blog/vc-or-bootstrap.md

```diff
@@ -0,0 +1,101 @@
+
+Of course, this can easily result in a lack of discipline. However, it has helped us be able to get an open source project off the ground first, before focussing on revenue last summer.
+
+## Is fundraising a drag?
+
+There is a huge amount of variation. It all comes down to roughly these requirements:
+
+* clear explanation of what you're working on
+* fast progress being made
+* an idea that could be very big
+* proven team
+* good vc market conditions
+
+I've intentionally not listed "a good network" as, whilst easier for some than others, I believe you can get this done as a byproduct of the above.
+
+None of these are necessary, but the better you do against each of them, the less time you'll have to spend fundraising. Ironically, this means you'll have more time to focus on product, which means - the easier time you'll have fundraising.
+
+## More bosses
+
+Although you may retain control, if you fundraise, you've now got investors that you feel you _should_ report to.
+
+However, if you're building a bootstrapped company, you _still_ have to report to someone - your own team, if applicable, and your customers.
+
+One more party has made a positive difference to us, but for others it can be bad - if your board asks for X, and users want Y, then you may have problems. This depends on the kind of people you raise from and your relationship with them.
+
+There is a critical boss that VC removes though - you, to a certain extent. Personal finances can survive much longer when you have VC funding and when you're early stage. This can lead to a lack of discipline, as mentioned before, or it can lead to longer term thinking. It certainly feels less stressful. The fact you still need to be accountable for spending to your investors, we've found, does mean that when we spend a lot, it is intentional and reasoned.
+
+For example, we pay 4 months notice if we let someone go. I doubt we'd have made that call if we were bootstrapped. We simply think it's cheaper to spend money here than to have an underperformer stay for longer because we feel bad. 
+
+## How to make loads of money
+
+There's a myth that you can only exit (sell your shares) from a VC backed company through an Initial Public Offering (IPO) or through selling the business.
+
+The reality is that you can sell secondaries, but they aren't guaranteed. Most founders we know do this around the series B, or whenever the company is an obvious success. For employees, this normally starts a couple of rounds later. You can't sell anything like 100% of your stake each time - a few percent of what you own is the norm - VCs want you to remove some personal stress so you can focus on the business, but they still want you hungry for more. As the company gets bigger, this can go from removing any personal debt, to a life changing amount _whilst_ still running the company.
```

Comment:
```suggestion
The reality is that you can sell secondaries, but they aren't guaranteed. Most founders we know do this around the series B, or whenever the company is an obvious success. For employees, this normally starts a couple of rounds later. 

You can't sell anything like 100% of your stake each time – a few percent of what you own is the norm – but VCs want you to remove some personal stress while keeping you hungry for more. As the company gets bigger, this can go from removing any personal debt, to a life changing amount _whilst_ still running the company.
```